### ISSUE TITLE
Question about sample_sparse_structure

### ISSUE THREAD

**Issue by kevinhuangxf** [microsoft/TRELLIS#153]
Thanks for the excellent work!

I have a question about the sample_sparse_structure function. If it samples difference size of coordinates from the image encoded latents, how can this operation be applied for multiple batch size during training ?

**Comment by JeffreyXiang** [comment]
Hi,
Batched attention with different sequence length can be efficiently handled with modern attention implementation like `flash-attn` and `xformers`. See [the code](https://github.com/microsoft/TRELLIS/blob/eeacb0bf6a7d25058232d746bef4e5e880b130ff/trellis/modules/sparse/attention/full_attn.py#L189) here for more details.

**Comment by HuynhNguyenPhuc** [comment]
> Thank you! I have one more question. How is batch training set up about SparseTensors(feat=[k,1024],coords=[k,4])? (1) When I set batchsize=8, will the SparseTensors be (feat=[8_k,1024],coords=[8_k,4])? (2) If I need to pad the 8 sparsetensor to the same dim.?

@JeffreyXiang I also questioned about it too. Can you help me with this?
